zkevm-circuits/src/mpt_circuit.rs

```diff
@@ -870,6 +874,73 @@ impl<F: Field> MPTConfig<F> {
                     offset += 1;
                 }
 
+                // Rlp prefixes table [rlp_tag, byte, is_string, is_short, is_verylong]
+                for ind in 0..=127 {
```

Comment:
Could you do this a bit more like in the witness generation with a match statement for example. Will be quite a bit shorter, less duplicated code (like setting the tag to the same value for each case) and that also uses the RLP constants like `RLP_SHORT` etc... so fewer magic values.